caddyhttp/httpserver/server.go

```diff
@@ -406,3 +488,14 @@ func WriteTextResponse(w http.ResponseWriter, status int, body string) {
 	w.WriteHeader(status)
 	w.Write([]byte(body))
 }
+
+// HandleCommonError checks if the given error is an error that could pop up
+// when handling request. Currently only checks for MaxBytesReader error.
+// Returns a boolean - whether the given error is a common error, and a int -
+// the suggested HTTP status code to return to the user
+func IsCommonError(err error) (bool, int) {
+	if _, ok := err.(MaxBytesExceeded); ok {
+		return true, http.StatusRequestEntityTooLarge
+	}
+	return false, -1
```

Comment:
My intention is that the status code is advisory only, and is only applicable when the returned bool is `true`. It should be discarded if it is `false`.